plugins/pom-analyzer/src/main/java/eu/f4sten/pomanalyzer/data/ResolutionResult.java

```diff
@@ -38,43 +35,32 @@
     public File localPomFile;
 
     public ResolutionResult(String coordinate, String artifactRepository) {
+        validate(coordinate);
```

Comment:
I would suggest that we have a central place for handling all Maven-related stuff something like "MavenUtils?" Downloading Poms, Jars, checking if they exist or not, deriving their path, etc. would be implemented there and everyone else would use them. Right now if I implement the central .m2 feature in OPAL I have to implement similar things that you have already implemented here.